pkgs/development/libraries/glib/default.nix

```diff
@@ -75,8 +75,13 @@ stdenv.mkDerivation rec {
   # internal pcre would only add <200kB, but it's relatively common
   configureFlags = [ "--with-pcre=system" ]
     ++ optional stdenv.isDarwin "--disable-compile-warnings"
-    ++ optional (stdenv.isFreeBSD || stdenv.isSunOS) "--with-libiconv=gnu"
-    ++ optional stdenv.isSunOS "--disable-dtrace";
+    ++ optional (stdenv.isFreeBSD || stdenv.isSunOS || (stdenv.hostPlatform.libc != "glibc")) "--with-libiconv=gnu"
```

Comment:
I think `stdenv.hostPlatform.libc != "glibc"` covers the rest

Reply:
I thought so but didn't want to get that wrong... :).